libraries/client/src/main/java/org/holochain/androidserviceruntime/client/HolochainServiceIntentActions.kt

```diff
@@ -0,0 +1,15 @@
+package org.holochain.androidserviceruntime.client
+
+object HolochainServiceIntentActions {
```

Comment:
Move the intent action types from the service lib to the client lib, so that client apps can create intents with the proper action type.